Readme.md

```diff
@@ -23,11 +23,13 @@ For more information run `rpiboot -h`.
 Clone this repository on your Pi or other Linux machine.
 Make sure that the system date is set correctly, otherwise Git may produce an error.
 
-**This git repository uses symlinks. For Windows builds clone the repository under Cygwin**
+* This git repository uses symlinks. For Windows builds clone the repository under Cygwin.
+* Instead of duplicating the EEPROM binaries and tools the rpi-eeprom repository
+  is included as a (git submodule)[https://git-scm.com/book/en/v2/Git-Tools-Submodules]
 
 ```
 sudo apt install git libusb-1.0-0-dev pkg-config
-git clone --depth=1 https://github.com/raspberrypi/usbboot
+git clone --recurse-submodules --depth=1 https://github.com/raspberrypi/usbboot
```

Comment:
Does --depth=1 apply to submodules or do you have to include --shallow-submodules ?

Reply:
--depth=1 applies to 'update' but I think you're right about the clone command. It needs --shallow-submodules to pass --depth=1 to the submodules if --recurse-submodules is specified.